addon/components/au-dropdown.js

```diff
@@ -16,6 +16,23 @@ export default class AuDropdown extends Component {
     this.dropdownOpen = false;
   }
 
+  @action
+  toggleDropdown() {
+    this.dropdownOpen = !this.dropdownOpen;
+  }
+
+  @action
+  clickOutsideDeactivates(event) {
+    let toggleButton = document.querySelector('[data-au-dropdown-toggle]');
```

Comment:
We can replace this with a local modifier so we don't need to add the data attribute, but that requires Ember 3.25+ so we can only do that in the v2 release.